tests/integration/_experimental/object_detection/test_workflow.py

```
@@ -15,22 +15,24 @@
 
 import pytest
 
-from kolena._experimental.object_detection import GroundTruth
-from kolena._experimental.object_detection import Inference
-from kolena._experimental.object_detection import Model
-from kolena._experimental.object_detection import TestCase
-from kolena._experimental.object_detection import TestSample
-from kolena._experimental.object_detection import TestSuite
 from kolena.workflow import test
 from kolena.workflow.annotation import LabeledBoundingBox
 from kolena.workflow.annotation import ScoredLabeledBoundingBox
 from tests.integration.helper import fake_locator
 from tests.integration.helper import with_test_prefix
 
+object_detection = pytest.importorskip("kolena._experimental.object_detection")
```

In which case are we expecting these imports to fail?

---

This is dependant on the CI environment having the extra `kolena[metrics]`. If it doesn't have it, imported functions fail when seeing `sklearn_metrics.function`.

---

Ah I see. Thanks for the explanation.

---

If our CI test environment doesn't have `kolena[metrics]` installed, will it silently skip the test?
Curious if there is a better workaround that doesn't silently skip tests. CC @munkyshi